.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/fmtlib/fmt.git
 [submodule "extlib/HighFive"]
 	path = extlib/HighFive
-	url = https://github.com/BlueBrain/HighFive.git
+	url = https://github.com/openbraininstitute/HighFive.git
```

Comment:
any reason why we're not using the new official one?
https://github.com/highfive-devs/highfive

Reply:
yes! I had no idea what was the new official one. I just guessed